app/modules/sightings/models.py

```diff
@@ -1073,6 +1075,12 @@ def _ensure_annot_data_in_response(self, annot, response):
                 'image_url': annot.asset.get_image_url(),
                 'asset_dimensions': annot.asset.get_dimensions(),
                 'bounds': annot.bounds,
+                'sighting_guid': self.guid,
+                'sighting_time': self.get_edm_data_field('time'),
+                'sighting_time_specificity': self.get_edm_data_field('timeSpecificity'),
+                'encounter_guid': annot.encounter.guid if annot.encounter else None,
+                'asset_filename': annot.asset.filename,
+                'individual_first_name': individual.names[0] if individual else None,
```

Comment:
`names[0]` is a Name (class) object.  luckily someone (not me) seems to have made `indiv.get_first_name()` which does exactly the right thing you would need here!  (including handling the case where there is none.)